parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2103,21 +2103,19 @@ toSlurpResult currentPath uf existingNames =
   updates = SlurpComponent (Set.fromList types) (Set.fromList terms) where
     terms =
       [ var n
-      | (n, r'@Referent.Ref{}) <- R.toList (Names.terms fileNames0)
-      , [r@Referent.Ref{}]     <- [toList $ Names.termsNamed existingNames n]
-      , r' /= r
+      | (n, Referent.Ref{}) <- R.toList (Names.terms fileNames0)
+      , [Referent.Ref{}]     <- [toList $ Names.termsNamed existingNames n]
       ]
     types =
       [ var n
-      | (n, r') <- R.toList (Names.types fileNames0)
-      , [r]     <- [toList $ Names.typesNamed existingNames n]
-      , r' /= r
+      | (n, _) <- R.toList (Names.types fileNames0)
+      , [_]    <- [toList $ Names.typesNamed existingNames n]
```

Comment:
I think we want to revert this change.  if `(n, r') == (n, r)`, then it's an no-op; it belongs in `dups` (and I assume it's already there).